### PR TITLE
Handle 'Classification outcome' cohort filter for multiclass scenario in SDK

### PR DESCRIPTION
**Pull request by gaugup**
## Description

It looks like we have "Classification Error" (aka "Classification outcome") filter supported for multiclass scenario. This was added as a part of the PR https://github.com/microsoft/responsible-ai-toolbox/pull/1371. Hence, supporting the "Classification outcome" filter for multiclass case. 

## Checklist

- [ ] I have added screenshots above for all UI changes.
- [ ] I have added e2e tests for all UI changes.
- [ ] Documentation was updated if it was needed.

### CODE REVIEW

File: erroranalysis/tests/test_cohort_filter.py [hunk 1]
```diff
@@ -137,6 +137,40 @@ def test_cohort_filter_in_the_range_of(self):
...
+    def test_cohort_filter_multiclass_classification_outcome(
+            self, arg, correct_prediction):
+        X_train, X_test, y_train, y_test, feature_names = create_iris_pandas()
+        model = create_sklearn_svm_classifier(X_train, y_train)
+        model_task = ModelTask.CLASSIFICATION
+        categorical_features = []
+
+        # the index 1, corresponds to incorrect prediction
+        # the index 0 correspond to correct prediction
+        filters = [{'arg': arg,
```
Comment (imatiach-msft) [inline erroranalysis/tests/test_cohort_filter.py:151]: I thought I had constants for all of these - arg, column, method

Comment (imatiach-msft) [reply erroranalysis/tests/test_cohort_filter.py:151]: ah yes, there is:
```
from erroranalysis._internal.constants import (ARG, COLUMN, METHOD)
```
but it looks like the other tests also need to be updated in this file to use them

**Review by imatiach-msft — APPROVED** [review]

File: erroranalysis/tests/test_cohort_filter.py [hunk 2]
```diff
@@ -275,5 +309,5 @@ def run_error_analyzer(validation_data,
                                        composite_filters)
 
     # validate there is some data selected for each of the filters
-    assert validation_data.shape[0] > 0
+    assert validation_data.shape[0] >= 0
```
Comment (imatiach-msft) [inline erroranalysis/tests/test_cohort_filter.py:312]: This means the filter was so strict that all data was filtered out, if ```validation_data.shape[0] == 0```  - was there a specific test which this was the case for?  I'm just worried some code might change and our filtering breaks to filter everything out.  Perhaps, if this is needed, we can have a flag for the == 0 case?

File: erroranalysis/tests/test_cohort_filter.py (hunk 1 above)
Comment (gaugup) [reply erroranalysis/tests/test_cohort_filter.py:151]: Yeah, I have create a task internally to migrate these to module constants.

File: erroranalysis/tests/test_cohort_filter.py (hunk 2 above)
Comment (gaugup) [reply erroranalysis/tests/test_cohort_filter.py]: The multiclass classification model was giving exactly the same predictions as the test labels. That's why I had to update the assert condition. I am not passing a flag for cases when we want to assert on zero.

**Pull request merged**